CMakeLists.txt

```diff
@@ -134,20 +138,19 @@ get_target_property_required(YAPF env YAPF)
 add_custom_target(
   check_python
   DEPENDS conda_yapf
+  COMMAND ${symbiflow-arch-defs_SOURCE_DIR}/common/python_health_helper.sh -c -y ${YAPF}
   )
 add_custom_target(
   format_python
   DEPENDS conda_yapf
+  COMMAND ${symbiflow-arch-defs_SOURCE_DIR}/common/python_health_helper.sh -f -y ${YAPF}
   )
 
-add_custom_command(
-  TARGET check_python
-  COMMAND ${symbiflow-arch-defs_SOURCE_DIR}/common/yapf_helper.sh -c -y ${YAPF}
-  )
-
-add_custom_command(
-  TARGET format_python
-  COMMAND ${symbiflow-arch-defs_SOURCE_DIR}/common/yapf_helper.sh -f -y ${YAPF}
+get_target_property_required(FLAKE8 env FLAKE8)
+add_custom_target(
+  lint_python
+  DEPENDS conda_flake8
```

Comment:
Rather than depending on ``conda_flake8`` directly, use ``get_target_property(FLAKE8_TARGET env FLAKE8_TARGET)``.  Directly depending on ``conda_XXX`` is part of the cause of https://github.com/SymbiFlow/symbiflow-arch-defs/issues/564

Reply:
Done